frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/tests/premium.unit.spec.tsx

```diff
@@ -54,7 +54,7 @@ describe("Static Embed Setup phase - EE, with token", () => {
 
         await userEvent.click(fontSelect);
 
-        const popover = await screen.findByRole("grid");
+        const popover = await screen.findByRole("listbox", { name: "Font" });
```

Comment:
We used the old `Select` component which renders different content, now we use Mantine's `Select` so I need to change the query to match the new select popover.